packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Models/Serialization/Json/JsonObjectSerialization.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Generator.CSharp.ClientModel
+{
+    internal sealed class JsonObjectSerialization
```

Comment:
I don't think we need this class when we get to expressions only.

Reply:
Yeah I think so as well

Reply:
I'll remove it.